functions/src/Services/Elections.ts

```diff
@@ -11,4 +13,18 @@ export const openElections = () => {
 		})
 		.then(() => Promise.resolve())
 		.catch(error => Promise.reject(error));
+};
+
+export const addPosition = (position: Position) =>{
+	const electionsCollection = getElectionsCollection();
+	const appendPosition = firestore.FieldValue.arrayUnion({ ...position });
```

Comment:
instead of firestore could u use db. if it doesn't work u can just use firestore i want to keep our db styling consistent through our code base.

Reply:
I can't use db for this one, I tested it, but FieldValue seems to only work with firestore